File: LeetCode/LeetCode7.py
class Solution(object):
    def reverse(self, x):
        """
        :type x: int
        :rtype: int
        """
        if abs(x)>2147483648:return 0
        fuhao=1
        if x<0:
        	fuhao=-1
        	x=-x
        y=temp=0
        while x>0:
        	y*=10;
        	temp=x%10;
        	x/=10;
        	y=y+temp;
        	if y>2147483648:return 0
        return y*fuhao
x=-787465
t=Solution()
print (t.reverse(x))   

File: LeetCode/LeetCode7.cpp
class Solution {
public:
    int reverse(int x) {
    	int y=0;
    	int temp=0;
        while(x>0)
        {
        	y*=10;
        	temp=x%10;
        	x/=10;
        	y=y+temp;
        }
        return y
    }
    int main(int argc, char const *argv[])
    {
    	/* code */
    	int x=123;
        t=new Solution();
    	printf("%d\n",t.reverse(x));
    	return 0;
    }
};